romana-install/romana-setup

```diff
@@ -49,14 +56,53 @@ if (( $# > 0 )); then
 				usage
 				exit 0
 				;;
-			-t|--target)
+			-n|--name)
+				case "$2" in
+					[a-zA-Z][a-zA-Z0-9]*)
+						stack_name="$2"
+						shift 2
+						;;
+					*)
+						echo "Invalid stack name: '$2' (must start with a letter, and contain only letters and digits)"
```

Comment:
Length restrictions: there will be. I don't know which part would be the limit yet -- could be cloudformation, vagrant, virtualbox, etc.
Case-sensitive: I don't think so, but also worth testing.